### PR TITLE
Util/logs

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys, I added "log4J" for our logs and to be able to use it in our classes, please check it out.

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+log4j.appender.FileAppender.File=log/logSearch.log
+log4j.appender.HTML.File=./log/application.html
+
+# Here we define the layout and pattern
+log4j.appender.FileAppender.DatePattern='.'MM-dd-yyyy
+log4j.appender.FileAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.FileAppender.layout.ConversionPattern= %d{HH:mm:ss} %-5p [%C{1}][%M][%L] %m%n
+log4j.appender.HTML.layout=org.apache.log4j.HTMLLayout
+log4j.appender.HTML.layout.Title=Application log
+log4j.appender.HTML.layout.LocationInfo=true
```
Comment (JosueRodriguezGarcia) [inline src/main/resources/log4j.properties]: add line.

File: src/main/java/com/foundation/trello/util/Log.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        if (instance == null) {
+            instance = new Log();
+            PropertyConfigurator.configure(LOG4J_PROPERTIES);
+            log.setLevel(Level.ALL);
+        }
+        return instance;
+    }
+
+    /**
+     *This method returns a 'log' with which more loggers can be added.
```
Comment (JosueRodriguezGarcia) [inline src/main/java/com/foundation/trello/util/Log.java]: add space between "*" and "This"

File: src/test/java/com/foundation/trello/model/ReadConfigurationTest.java [hunk 3]
```diff
@@ -99,36 +125,39 @@ public void putRequestName() {
...
+        Log.getInstance().getLog().info("Status Code: " + response.getStatusCode());
     }
 
     /**
      * This test verifies that the DELETE return a state 200.
      */
     @Test
     public void deleteRequest() {
-        given().
+
```
Comment (JosueRodriguezGarcia) [inline src/test/java/com/foundation/trello/model/ReadConfigurationTest.java]: delete line

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (MadayAlcala) [reply src/main/resources/log4j.properties]: Solved.

File: src/main/java/com/foundation/trello/util/Log.java (hunk 2 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/util/Log.java]: Solved.

File: src/test/java/com/foundation/trello/model/ReadConfigurationTest.java (hunk 3 above)
Comment (MadayAlcala) [reply src/test/java/com/foundation/trello/model/ReadConfigurationTest.java]: Solved.

**Review by JosueRodriguezGarcia — APPROVED** [review]

File: src/main/java/com/foundation/trello/util/Log.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+/**
+ * This class is used for testing 'ReadConfiguration' class.
+ *
+ * @author Maday Alcalá Cuba.
+ * @version 0.0.1
+ */
+public final class Log {
+
+    private static final String LOG4J_PROPERTIES = "src/main/resources/log4j.properties";
+
```
Comment (raulchoque-eng) [inline src/main/java/com/foundation/trello/util/Log.java]: delete the white space please

File: src/main/java/com/foundation/trello/util/Log.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    private static Log instance = null;
+
```
Comment (raulchoque-eng) [inline src/main/java/com/foundation/trello/util/Log.java]: delete the white space please

File: src/main/java/com/foundation/trello/util/Log.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    /**
+     * Private constructor for the Singleton pattern.
+     */
+    private Log() {
+        super();
+    }
+
+    /**
+     * This method returns a log instance.
```
Comment (raulchoque-eng) [inline src/main/java/com/foundation/trello/util/Log.java]: Be more specific please

File: src/main/java/com/foundation/trello/util/Log.java [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+            log.setLevel(Level.ALL);
+        }
+        return instance;
+    }
+
+    /**
+     * This method returns a 'log' with which more loggers can be added.
+     * @return a 'log'.
+     */
+    public org.apache.log4j.Logger getLog() {
```
Comment (raulchoque-eng) [inline src/main/java/com/foundation/trello/util/Log.java]: This import have to go in class's Head please

File: src/main/java/com/foundation/trello/util/Log.java (hunk 4 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/util/Log.java]: Solved.

File: src/main/java/com/foundation/trello/util/Log.java (hunk 5 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/util/Log.java]: Solved.

File: src/main/java/com/foundation/trello/util/Log.java (hunk 6 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/util/Log.java]: Solved.

File: src/main/java/com/foundation/trello/util/Log.java (hunk 7 above)
Comment (MadayAlcala) [reply src/main/java/com/foundation/trello/util/Log.java]: Solved.

**Review by raulchoque-eng — APPROVED** [review]

**Pull request merged**